.github/scripts/templates/plugin_issues.md.jinja

```diff
@@ -0,0 +1,37 @@
+%% do NOT edit this file directly, only edit the template file %%
+
+We have collected a number of issues that are a great for getting started and helping with the development
+and documentation of plugins.
+
+The following lists are automatically generated, so some descriptions might be a bit cryptic.
+
+## Good first issue
+
+The following issues are good for beginners wanting to help with fixing issues and adding features or documentation.
+
+{% for issue in good_first_issue %}
+- [[{{ issue.pluginID }}|{{ issue.plugin }}]] - [{{ issue.title }}]({{ issue.url }})
+{% endfor %}
+
+## Help wanted
+
+The plugin developer is looking for help with these issues.
+This may be a result of a number of factors:
+- wanting to introduce newcomers to the project
+- the inability to properly address the issue/implement the feature request
+- or simple time constraints
+- and many more
+These issues might not always be easy to solve.
+
```

Comment:
When this list is viewed in the generated file, there is a blank line between each issue...

This should be fixed by adding a blank line between lines 23 and 24...